### PR TITLE
[Doc Improvement][Update existing code snippet]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b332592852ef3c1df31f0cd6e349854699eace29/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 91 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be5b2c86-6c1d-46aa-965d-a8f7e53dd451) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3d7c76d3b6f7d8e7a127d696fcc34c51a1a68dd4/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9de886c-20ed-41b2-822f-6640251d8dbe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/83d0ad032ef08032508e265177ae6d4f621824a4/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/758724df-38fc-4e03-841b-44f7ba89e998) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08bfb00c823d519d90168405db77ad3b399e09f3/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/204ce4e9-bee2-4fce-a97d-291931e0cd46) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a6b9e6812169ed266d6119b61827b127da29cb99/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/661574bd-18d2-470e-8a2a-925de71503d0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/79f759a635c0ebef57f478d94789b673843c78ce/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3eb8f1f7-9da1-4765-9add-e59328a38df9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a19bc5d7e7f66bb32f07ca70251f0c4a715ac734/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/86a503a1-bc6b-406e-98d4-29bdf96abdf6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/887772a374bb0f696048dcb7d76418bb3ab1aafd/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb483504-08c8-40fe-8cbf-dd401869815e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a2a0271c39314cac67118e2f8739ef26cb5eb9b5/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/143eca72-112c-4afd-b5ce-11a0c93b8354) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f5f478f5005c7ef18f7ce48d51c6abc7f7e124c/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78d292e7-1755-48fa-b977-ea6c237baa35) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7ca58815b5dc16e01d5a8ef983d075d719735a0f/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6fdbdfb7-2d65-46e9-88be-178c791b6077) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md [hunk 1]
```diff
@@ -47,33 +47,55 @@ To enable in-app purchase experience, update your Teams app `manifest.json` file
 
 To trigger in-app purchase for the app, invoke the `openPurchaseExperience` API from your web app.
 
-Following is an example of calling the API from the app:
+Following code snippet is an example of calling the API from the Teams app built using teams-js sdk up to version 11 and version 2:
+
+# [TeamsJS V11](#tab/jsonV11)
```
Comment (Wajeed-msft) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md:52]:
```suggestion
# [TeamsJS v1](#tab/jsonV11)
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md [hunk 2]
```diff
@@ -47,33 +47,55 @@ To enable in-app purchase experience, update your Teams app `manifest.json` file
 
 To trigger in-app purchase for the app, invoke the `openPurchaseExperience` API from your web app.
 
-Following is an example of calling the API from the app:
+Following code snippet is an example of calling the API from the Teams app built using teams-js sdk up to version 11 and version 2:
```
Comment (Wajeed-msft) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md:50]:
```suggestion
Following code snippet is an example of calling the API from the Teams app built using Teams JavaScript client SDK:
```

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0002f8a54d83c888b7669078eeed56867f137e2b/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a66b8af7-63fe-4d64-bd23-535726d8f3c0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/in-app-purchase-flow.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0002f8a54d83c888b7669078eeed56867f137e2b/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fappsource%2Fprepare%2Fin-app-purchase-flow.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5966adb-56c7-4871-8466-d069f8002405) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**